webpages/settings/components/addon-setting.html

```diff
@@ -38,11 +38,15 @@
       type="text"
       class="setting-input string"
       v-model="addonSettings[addon._addonId][setting.id]"
-      @input="updateSettings()"
+      @change="checkValidity() || updateSettings()"
       :disabled="!addon._enabled"
       v-if="setting.type === 'string'"
       :placeholder="setting.default"
-      maxlength="100"
+      :maxlength="setting.max || 100"
+      :minlength="setting.min || 0"
```

Comment:
Default should probably be 1?

Reply:
Maybe, not high prio as discuss button is the only use for this